src/contexts/Photos/Photos.context.tsx

```diff
@@ -314,14 +350,15 @@ export const PhotosContextProvider: React.FC = ({ children }) => {
     await startPhotos();
   }
 
-  function handleRemovePhotosItems(photosItems: PhotosItem[]) {
-    syncedPhotosItems.current = syncedPhotosItems.current.filter((mergedPhotosItem) =>
-      photosItems.find((pi) => pi.name === mergedPhotosItem.name) ? false : true,
-    );
+  async function handleRemovePhotosItems(photosItems: PhotosItem[]) {
+    for (const photosItem of photosItems) {
+      if (photosItem.photoId) {
+        await photos.database.deleteSyncedPhotosItem(photosItem.photoId);
```

Comment:
Maybe a bulk delete could be a good improvement on speed/resources than this, just using:

```
const photosWithId = photosItems.filter((photo) => !!photo.photoId);
for (let i = 0; i < photosWithId.length; i += 20) {
    await photos.database.deleteSyncedPhotosItems(nPhotos.slice(i, i+20));
}
```

Reply:
yep, should be done using a transaction, need to go deeper into that